docs/admin-tutorial.md

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
Duplicate page for `admin-api-tutorial.md`. Also the `id` is conflict.

Pushing a follow-up for removing this page...

Reply:
@tisonkun Oops! Thanks for your eagle eye!